### PR TITLE
Add doc of Open Graph/Twitter image param

### PR DESCRIPTION
**Pull request by Sp-cy**
This image will be used when sharing your site in social media or messaging apps. Value can be a link. If it's a relative link it seems to work best if you point out the internal subfolder within your static files directory like ["img/yesnoyes.png"]. 
It is also required for the values to be inside [""] brackets and you need to add front matter values like "images" "video" in your arquetypes to populate the twitter and open graph meta tags. Values in front matter should also be like config file values within [""]. View HUGO documentation on Internal Templates for more info.

### CODE REVIEW

**Comment by g1eny0ung** [comment]
LGTM! I will follow this PR to further improve related docs.

**Comment by Sp-cy** [comment]
For the front matter parameters I've found that it's more convenient to use - Listed items for the link values instead of my initial proposal. This should look like the categories and tags listed values.
```
- https://github.com/
- https://gist.github.com/
```
 NetlifyCMS goes a bit crazy and it's more complicated in my initial manner, this is not a problem and multiple values can be added with comma separation. If links are - Listed values the parameter seem to still populate the open graph and twitter values appropiately. It's worth stating that open graph only accepts absolute links by specification. But somehow HUGO knows how to report the Config file parameter from a relative path.
 **In summary: Twitter and open graph links in front matter should be absolute. And they should be stated as listed values.**
